### ISSUE TITLE
DATEV Export

### ISSUE THREAD

**Issue by sweitmann** [ledger123/runmyaccounts#52]
Hi Armaghan,

The line tax accounts need to be added to our export file and we need to sort out the error row (same account in debit and credit)

Regards

**Comment by sweitmann** [comment]
Hi Armaghan,

I'm using a dataset booked through the master branch to test the new DATEV Export branch. All transactions in it were booked with the 'Line Tax' feature activated, so I was hoping that I could also use it to test the new DATEV Export branch. 

But I've noticed that we seem to have some small differences between the data structure in the master branch dataset and the DATEV Export branch dataset.

![bildschirmfoto vom 2016-06-27 15 24 14](https://cloud.githubusercontent.com/assets/100664/16381233/c63391a2-3c7b-11e6-8eef-7d7f629b292a.png)

Is there some kind of re-posting script or SQL statement that we can include in the 'Export--Prepare DATEV file' feature that can update my master branch 'Line Tax' dataset to make it compatible and usable for testing the DATEV Export branch?

![bildschirmfoto vom 2016-06-27 15 29 00](https://cloud.githubusercontent.com/assets/100664/16381253/e81c18d4-3c7b-11e6-91f5-aff32f091011.png)



Regards

**Comment by sweitmann** [comment]
Hi Armaghan,
 
I’m exporting some data using our “052_datev_export” branch and if I use the dropdown menu to enter the period, everything works fine. But when I try to enter a date in “From” and “To” instead, when I click on “Continue”, the date I entered in “From” changes from 01.09.2016 to 20.9..01.0 and the date I entered in “To” changes from 30.11.2016 to 20.1..30.1. I also get the following error message on my screen:

![bildschirmfoto 2017-04-24 um 08 42 40](https://cloud.githubusercontent.com/assets/100664/25325344/6ce951b2-28cb-11e7-9a00-cad9b0af22fa.png)

Any ideas?

Regards
